desktop/cmp/rest/regroup/RegroupDialogModel.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2020 Extremely Heavy Industries Inc.
+ */
+import {uniq} from 'lodash';
+import {HoistModel, XH} from '@xh/hoist/core';
+import {action, bindable, observable} from '@xh/hoist/mobx';
+
+
+@HoistModel
+export class RegroupDialogModel {
```

Comment:
Regroup is very specific to configs and prefs. This change should be in a separate PR and apply to only Config and Pref rather than included in the rest package and the action apply to rest grid.